#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_type_basic.cpp.dpatch by Christian Marillat <marillat@debian.org>
##
## All lines beginning with ## DP:' are a description of the patch.
## DP: 

@DPATCH@

--- mpeg4ip-1.6.orig/common/video/iso-mpeg4/src/type_basic.cpp	2007-09-22 15:31:16.000000000 +0200
+++ mpeg4ip-1.6/common/video/iso-mpeg4/src/type_basic.cpp	2007-09-22 15:31:26.000000000 +0200
@@ -317,7 +317,7 @@
 	iHalfY = m_vctTrueHalfPel.y - iMVY * 2;
 }
 
-Void CMotionVector::setToZero (Void)
+Void CMotionVector::setToZero ()
 {
 	memset (this, 0, sizeof (*this));
 }
